src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2115,42 +2240,56 @@
           short-summary: Port number on which to receive traffic.
         - name: --dest-address
           short-summary: The IP address or URI at which to receive traffic.
+    examples:
+        - name: This example checks connectivity to a destination virtual machine over port 80.
```

Comment:
Remove `This example`

Reply:
To be clear, you mean removing those specific words yes?  You could also see this as "Remove this example" which I would not recommend :)

Reply:
Addressed.